### PR TITLE
update ucla message handlers

### PR DESCRIPTION
**Pull request by emord**
@gcapalbo https://manage.dimagi.com/default.asp?248775

@benrudolph

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -1476,6 +1476,11 @@
     "FRI_SMS_SHIFT": "custom.fri.api.shift_custom_content_handler",
     "FRI_SMS_OFF_DAY": "custom.fri.api.off_day_custom_content_handler",
     "UCLA_MESSAGE_BANK": "custom.ucla.api.ucla_message_bank_content",
+    "UCLA_GENERAL_HEALTH": "custom.ucla.api.general_health_message_bank_content",
+    "UCLA_MENTAL_HEALTH": "custom.ucla.api.mental_health_message_bank_content",
+    "UCLA_SEXUAL_HEALTH": "custom.ucla.api.sexual_health_message_bank_content",
+    "UCLA_MED_ADHERENCE": "custom.ucla.api.med_adherence_message_bank_content",
+    "UCLA_SUBSTANCE_ABUSE": "custom.ucla.api.substance_use_message_bank_content",
```
Comment (emord) [inline settings.py]: Should this be standardized to substance_abuse or substance_use ?

File: settings.py [hunk 2]
```diff
@@ -1476,6 +1476,11 @@
     "FRI_SMS_SHIFT": "custom.fri.api.shift_custom_content_handler",
     "FRI_SMS_OFF_DAY": "custom.fri.api.off_day_custom_content_handler",
     "UCLA_MESSAGE_BANK": "custom.ucla.api.ucla_message_bank_content",
```
Comment (emord) [inline settings.py]: Your ticket didn't mention keeping this, but I assume we need to while they migrate their app?

File: settings.py (hunk 1 above)
Comment (gcapalbo) [inline settings.py]: sorry, substance_use for both

File: settings.py (hunk 2 above)
Comment (gcapalbo) [inline settings.py]: i should have said, we can get rid of it, it's only in the test phases right now so it's not actually being used

**Comment by gcapalbo** [comment]
looks good to me. could we also add the lookup table name to the error messages at the end of the api.py file under `if len(custom_messages) != 1:`

**Review by gcapalbo — APPROVED** [review]

**Pull request merged**